### PR TITLE
fix: Added styles for image section.

### LINKED ISSUE #167: Fix: ScaledImage styling
### Describe the bug
There is a `ScaledImage` component which is used to render images. It doesn't take any style prop, but it's expected when used to create `ImageContent` and `ImageTextSection` components. This results in expected styles like padding ignored.

### To Reproduce
1. Go to the homepage
2. Scroll down to a section with image on right
3. Observe that there is no (expected) space between the text and the image.

### Expected behavior
Either the `ScaledImage` component should take and handle `style` prop or the styling should be explicitly added to its parent.

### Screenshots
image

### PR DESCRIPTION
**Pull request by nlok5923**
### Description
Added styling for scaled image component by wrapping inside view to tackle the cases when image renders on right side.

Fixes #167 

### Type of Change:
**Delete irrelevant options.**

- Code
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
If you are working on a design then add a gif to show the responsiveness of your design.
Please follow [testing guidelines](https://github.com/anitab-org/anitab-org.github.io/wiki/Prepare-testing-GitHub-Pages-for-your-PR) and share the link of deployed site here. 
link : https://xenodochial-noether-fee8c5.netlify.app/

### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have attached link of deployed site.

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/Components/ImageContent/index.js [hunk 1]
```diff
@@ -26,11 +26,13 @@ const ImageContent = ({
         }
       }}
     >
+    <View style={styles.imageSection}>
```
Comment (tichnas) [inline src/Components/ImageContent/index.js:29]: Change `styles.imageSection` to `styles.sectionImage`

File: src/Components/ImageContent/index.js [hunk 2]
```diff
@@ -26,11 +26,13 @@ const ImageContent = ({
...
       <ScaledImage
         source={image}
         width={imageContainerWidth}
         style={styles.sectionImage}
```
Comment (tichnas) [inline src/Components/ImageContent/index.js:33]: remove this

File: src/Components/ImageContent/index.js [hunk 3]
```diff
@@ -59,6 +61,11 @@ const createStyles = (imageSide) =>
       paddingLeft: 16,
       paddingRight: 16,
     },
+    imageSection: {
```
Comment (tichnas) [inline src/Components/ImageContent/index.js:64]: no need for `imageSection` style

**Review by tichnas — CHANGES_REQUESTED** [review]
Thanks for the PR :smiley: 
I've requested some changes in `ImageContent` so also do them in the other file. After that everything's perfect.

**Comment by nlok5923** [comment]
Thanks @tichnas for requesting changes i had done the said changes please review the pr.

**Review by tichnas — APPROVED** [review]

**Review by tichnas — CHANGES_REQUESTED** [review]
@nlok5923 Really sorry to request changes again. Just found a small problem in responsiveness with the code.
In the `ImageContent` component, we have to remove `flex: 1` from `styles.sectionImage` (line 58), otherwise responsiveness breaks.

**Comment by nlok5923** [comment]
@tichnas it's absolutely fine. I had done the changes requested please review the pr.
Thanks,

**Review by tichnas — APPROVED** [review]

**Review by annabauza — APPROVED** [review]

**Pull request merged**